bench/workgen/workgen_int.h

```diff
@@ -198,14 +200,24 @@ struct TableRuntime {
 };
 
 struct ContextInternal {
+    // Dedicated to tables that are alive until the workload ends.
     std::map<std::string, tint_t> _tint;           // maps uri -> tint_t
     std::map<tint_t, std::string> _table_names;    // reverse mapping
     TableRuntime *_table_runtime;                  // # entries per tint_t
     uint32_t _runtime_alloced;                     // length of _table_runtime
     tint_t _tint_last;                             // last tint allocated
+
+    // Dedicated to tables that can be created or removed during the workload.
+    std::map<std::string, tint_t> _dyn_tint;
+    std::map<tint_t, std::string> _dyn_table_names;
+    std::vector<TableRuntime> *_dyn_table_runtime;
+    tint_t _dyn_tint_last;
+
     // unique id per context, to work with multiple contexts, starts at 1.
     uint32_t _context_count;
 
+    std::mutex* _mutex;
```

Comment:
Please add a comment about what this mutex is protecting. Also "_mutex" sounds too generic. Maybe call it "_dyn_tables_mutex" or something of that order.

Reply:
I agree, `mutex` is way too generic. I updated it to `_dyn_mutex`, let me know what you think: 1fc78d9916c76608c2b9ec94f7785e91ae16fb31